profiler/kokoro/integration_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+retry() {
+  for i in {1..3}; do
+  [ $i == 1 ] || sleep 10  # Backing off after a failed attempt.
+    "${@}" && return 0
+  done
+  return 1
+}
+
+# Fail on any error.
+set -eo pipefail
+
+# Display commands being run.
+set -x
+
+cd $(dirname $0)/..
+
+export GOOGLE_APPLICATION_CREDENTIALS="${KOKORO_KEYSTORE_DIR}/72935_cloud-profiler-e2e-service-account-key"
+export GCLOUD_TESTS_GOLANG_PROJECT_ID="cloud-profiler-e2e"
+
+# Run test.
+retry go mod download
+retry go test -c
+./e2e.test -run_backoff_test=true
```

Comment:
I don't think this needs the compile step? You should be able to pass flags directly.

```suggestion
retry go test -run_backoff_test=true
```

> The 'go test' command rewrites or removes recognized flags, as appropriate, both before and after the optional package list, before invoking the test binary.

https://golang.org/cmd/go/#hdr-Testing_flags

Reply:
Thanks! Done.